internal/app/appreciation/service.go

```diff
@@ -42,75 +43,81 @@ func (apprSvc *service) CreateAppreciation(ctx context.Context, appreciation dto
 
 	//add quarter
 	appreciation.Quarter = utils.GetQuarter()
+	logger.Debug(ctx, "service: CreateAppreciation: appreciation: ", appreciation)
 
 	//add sender
 	data := ctx.Value(constants.UserId)
 	sender, ok := data.(int64)
 	if !ok {
-		logger.Error("err in parsing userid from token")
+		logger.Error(ctx, "service: err in parsing userid from token")
 		return dto.Appreciation{}, apperrors.InternalServer
 	}
```

Comment:
should create a method in contextstore and use it

Reply:
will create contextstore and methods in seperate PR